tests/network/primary_pod_network.go

```diff
@@ -82,6 +86,59 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 			It("should report PodIP as its own on interface status", func() { AssertReportedIP(vmi) })
 		})
 	})
+
+	Describe("Connectivity", func() {
+		const servicePort = 1500
+
+		AssertConnectivity := func(outboundVMI, inboundVMI *v1.VirtualMachineInstance) {
+			inboundVMIAddress := inboundVMI.Status.Interfaces[0].IP
+
+			By("Checking connectivity via ICMP")
+			Expect(tests.PingFromVMConsole(outboundVMI, inboundVMIAddress)).To(Succeed())
+
+			By("Checking connectivity via ICMP with MTU-sized frames")
+			mtu := getPodNetworkMTU(outboundVMI)
+			ipHeaderSize := 28 // IPv4 specific
```

Comment:
Please use a constant. Note that the name is misleading: There are 20 bytes of IP header + 8 bytes of ICMP header.
And the fact it is IPv4 should be embedded in the name.

I would also prefer to see the calculation and conversion which is not inline.